launchlib/config.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var (
-	subProcessNamePattern = regexp.MustCompile("^[a-z-]+$")
+	processNamePattern = regexp.MustCompile("^[a-zA-Z_-]+$")
```

Comment:
Why relax the regex?

Reply:
It looks like sls-packaging doesn't have a regex.  Happy to add this there as well, though that would technically be an sls-packaging break.

Reply:
Let's keep it at this original regex for now!